src/main/java/bwapi/Game.java

```diff
@@ -892,15 +889,12 @@ final public boolean isWalkable(final WalkPosition position) {
      * - 5: Very high ground doodad
      * .
      */
-    public int getGroundHeight(final int tileX, final int tileY) {
-        return getGroundHeight(new TilePosition(tileX, tileY));
+    final public int getGroundHeight(final int tileX, final int tileY) {
+        return isValidTile(tileX, tileY) ? groundHeight[tileX][tileY] : 0;
     }
 
-    public int getGroundHeight(final TilePosition position) {
-        if (!position.isValid(this)) {
-            return 0;
-        }
-        return groundHeight[position.x][position.y];
+    final public int getGroundHeight(final TilePosition position) {
+        return getGroundHeight(position.x, position.y);
```

Comment:
Allowing access to `getGroundHeight()` without having to allocate a TilePosition.